css/layout/flexbox.css

```diff
@@ -8,10 +8,15 @@
   flex-direction: column;
 }
 
-.flex-wrap {
+.is-wrap {
   flex-wrap: wrap;
 }
```

Comment:
flex-wrap sounds more meaningful than is-wrap

Reply:
Did this to shorten the name. can make it flex-wrap if it conflicts with some other class in the future.

Reply:
is-wrap sounds like a question, so flex-wrap sounds more meaningful